_posts/2020-03-18-cockpit-215.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 215
+author: kkoukiou
+date: 2020-03-18
+tags: cockpit linux
+slug: cockpit-215
+category: release
+summary: Cockpit with Network improvements and Docker partial deprecation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 215.
+
+### Networking: Show additional ports for each firewall zone
+
+The Firewall page now shows ports that are directly added to a zone right after the services list.
+![zone ports](/images/firewall-zone-ports.png)
```

Comment:
I think we should also mention that the firewalld page now [shows IDs instead of short names](https://github.com/cockpit-project/cockpit/pull/13738). @Gundersanne has the details. This should probably be its own header.

Reply:
Let's not do this, for most zones the difference should not be very user-visible.